Documentation/MRTKNuGetPackage.md

```diff
@@ -0,0 +1,66 @@
+# Mixed Reality Toolkit NuGet Package
+
+Mixed Reality Toolkit (MRTK) is now available as a NuGet package on NuGet.org. There are some differences when it comes to consuming NuGet version of MRTK as opposed to a .unitypackage, read [NuGet Package Considerations](###%20NuGet%20Package%20Considerations) below. If any issues are encountered, file an issue using this [template](https://github.com/microsoft/MixedRealityToolkit-Unity/issues/new?assignees=&labels=Bug,Package%20Management%20-%20NuGet&template=bug-report.md&title=).
```

Comment:
I've had issues whether these header hyperlinks don't resolve correctly for docfx, be sure to run MRTK docfx generation and view the output before commiting.

Reply:
Removed.